.github/workflows/build.yml

```
@@ -526,6 +526,22 @@ jobs:
       - run: cargo install taplo-cli --locked
       - run: taplo format --check
 
+  typos:
+    name: Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - run: cargo install typos-cli --locked
+      # NOTE: separate step seeems to be needed to check `.*.toml` - TODO NEEDS INVESTIGATION
```

in an enjoyable bit of irony I'm afraid I have to point out you have a typo here :laughing: 

```suggestion
      # NOTE: separate step seems to be needed to check `.*.toml` - TODO NEEDS INVESTIGATION
```

---

aargh

---

I have updated this proposal to now check all files known by git, with limited exceptions, should be able to avoid this kind of typo moving forward.